test/new-e2e/tests/containers/docker/docker_test.go

```diff
@@ -0,0 +1,141 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package docker for all kubernetes new E2E tests
+package docker
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	fakeintake "github.com/DataDog/datadog-agent/test/fakeintake/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/infra"
+	"github.com/DataDog/datadog-agent/test/new-e2e/tests/containers"
+	dockervm "github.com/DataDog/test-infra-definitions/scenarios/aws/dockerVM"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/stretchr/testify/suite"
+)
+
+type DockerSuite struct {
+	containers.BaseSuite
+	isDsdStandalone bool
+	stackName       string
+}
+
+func TestDockerSuite(t *testing.T) {
+	// Full Agent
+	suite.Run(t, &DockerSuite{stackName: "docker-stack", isDsdStandalone: false})
+
+	// DogstatsD Standalone
+	suite.Run(t, &DockerSuite{stackName: "docker-stack", isDsdStandalone: true})
+}
+
+func (suite *DockerSuite) getAgentImage() string {
+	if suite.isDsdStandalone {
+		return "datadog/dogstatsd:latest"
```

Comment:
Is the image name hardcoded? Will it be possible to run the tests on a different tag?